Jenkinsfile

```diff
@@ -10,9 +10,26 @@ pipeline {
                 sh 'cp /usr/local/etc/roms/mm.us.rev1.z64 baserom.mm.us.rev1.z64'
             }
         }
+        stage('Setup') {
+            steps {
+                sh 'make -j setup 2> tools/warnings_count/warnings_setup_new.txt'
+                sh 'cat tools/warnings_count/warnings_setup_new.txt'
```

Comment:
If this is a bash shell, it may be better to do `make -j setup 2> >(tee tools/warnings_count/warnings_setup_new.txt)` so that you don't have to wait until the second step to see the stderr. (Also I think if the first step *failed*, you wouldn't see stderr at all?)

Reply:
`tee` prints whatever it gets passed to, while writing to a file?

Reply:
Yes